bert_e/tests/test_bert_e.py

```diff
@@ -450,10 +510,37 @@ def test_branch_cascade_target_hotfix(self):
         fixver = ['6.6.6.3']
         self.finalize_cascade(branches, tags, destination, fixver)
 
-    def test_branch_cascade_invalid_dev_branch(self):
+    def test_branch_cascade_target_three_digit_dev(self):
+        """Test cascade targeting three-digit development branch"""
         destination = 'development/4.3.17'
         branches = OrderedDict({
-            1: {'name': 'development/4.3.17', 'ignore': False}
+            1: {'name': 'development/4.3', 'ignore': True},
```

Comment:
Shouldn't it be the following? 

```suggestion
            1: {'name': 'development/4.3', 'ignore': False},
```

Reply:
Indeed, dev/4.3.17 goes into dev/4.3